src/frontend-scripts/components/section-main/Gamechat.jsx

```diff
@@ -557,12 +557,19 @@ class Gamechat extends React.Component {
 				};
 			}
 		} else {
+			if ((user.wins || 0) + (user.losses || 0) < 10) {
+				return {
+					isDisabled: true,
+					placeholder: 'You must finish ten games to use observer chat'
+				};
+			}
+
 			if (
 				((gameInfo.general.disableObserver && gameInfo.general.disableObserverLobby) || gameInfo.general.private) &&
 				!(isStaff || (userInfo.isTournamentMod && gameInfo.general.unlisted))
 			) {
 				return {
-					isDisabled: true,
+					isDisabled: false,
```

Comment:
Is there a reason for this change and the other few? Is it just to allow commands to be used even if observer chat is disabled? -- It feels a little bit counterintuitive for the user is my only concern

Reply:
Yeah because line guessing is a command so it would be necessary to allow it in games where observer chat is disabled

Reply:
I think vig suggested it

Reply:
My only concern is that, as I mentioned, the box will now say "Observer chat is disabled" but still be available to type in -- it might be too long but in my mind I was thinking it would be something like "Observer chat is disabled but you can still use commands"